src/client.js

```diff
@@ -1,5 +1,5 @@
 var _ = require("lodash");
-var package = require("../package.json");
+var pkg = require("../package.json");
```

Comment:
This isn't for no reason. `package` is a reserved word in ES6, and webpack was refusing to compile this file without this rename.